docs/index.rst

```diff
@@ -1,240 +1,27 @@
-  Client <pubsub-client>
-  pubsub-topic
-  pubsub-subscription
-  pubsub-message
-  pubsub-iam
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Resource Manager
-
-  Overview <resource-manager-api>
-  resource-manager-client
-  resource-manager-project
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Runtime Configuration
-
-  runtimeconfig-usage
-  Client <runtimeconfig-client>
-  runtimeconfig-config
-  runtimeconfig-variable
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Spanner
-
-  spanner-usage
-  spanner-client-usage
-  spanner-instance-usage
-  spanner-database-usage
-  spanner-session-crud-usage
-  spanner-session-implicit-txn-usage
-  spanner-session-pool-usage
-  spanner-batch-usage
-  spanner-snapshot-usage
-  spanner-transaction-usage
-
-  spanner-client-api
-  spanner-instance-api
-  spanner-database-api
-  spanner-session-api
-  spanner-keyset-api
-  spanner-snapshot-api
-  spanner-batch-api
-  spanner-transaction-api
-  spanner-streamed-api
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Speech
-
-  speech-usage
-  Client <speech-client>
-  speech-encoding
-  speech-operation
-  speech-result
-  speech-sample
-  speech-alternative
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Stackdriver Error Reporting
-
-  error-reporting-usage
-  Client <error-reporting-client>
-  error-reporting-util
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Stackdriver Logging
-
-  logging-usage
-  Client <logging-client>
-  logging-logger
-  logging-entries
-  logging-metric
-  logging-sink
-  logging-stdlib-usage
-  logging-handlers
-  logging-handlers-app-engine
-  logging-handlers-container-engine
-  logging-transports-sync
-  logging-transports-thread
-  logging-transports-base
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Stackdriver Monitoring
-
-  monitoring-usage
-  Client <monitoring-client>
-  monitoring-metric
-  monitoring-resource
-  monitoring-group
-  monitoring-query
-  monitoring-timeseries
-  monitoring-label
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Storage
-
-  Client <storage-client>
-  storage-blobs
-  storage-buckets
-  storage-acl
-  storage-batch
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Translate
-
-  translate-usage
-  Client <translate-client>
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: Vision
-
-  vision-usage
-  vision-annotations
-  vision-batch
-  vision-client
-  vision-color
-  vision-crop-hint
-  vision-entity
-  vision-feature
-  vision-face
-  vision-image
-  vision-safe-search
-  vision-text
-  vision-web
-
-.. toctree::
-  :maxdepth: 0
-  :hidden:
-  :caption: External Links
-
-  GitHub <https://github.com/GoogleCloudPlatform/google-cloud-python/>
-  Issues <https://github.com/GoogleCloudPlatform/google-cloud-python/issues>
-  Stack Overflow <http://stackoverflow.com/questions/tagged/google-cloud-python>
-  PyPI <https://pypi.python.org/pypi/google-cloud>
+  :maxdepth: 2
+  :hidden:
+
+  core/index
```

Comment:
eventually all of the `usage.rst` will become `index.rst`. I'm trying to be minimal here.